.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__
 *.[aox]
 *.mod
 *.sw[a-p]
+.DS_Store
```

Comment:
Where are these two hidden directories coming from?

Reply:
(the second one being .idea, which was added in the last PR)

Reply:
.DS_Store is created if you open the folder on a macOS Finder

.idea/ is a folder that is specific to PyCharm. It is an IDE that stores user configuration for the IDE; like interpreter, virtual env. etc.